pkg/fileutil/fileutil.go

```diff
@@ -167,6 +168,11 @@ func CopyTree(src, dest string, uidRange *uid.UidRange) error {
 		return nil
 	}
 
+	fis, err := ioutil.ReadDir(dest)
```

Comment:
Alright, then I think it is the caller of `CopyTree` who should be responsible for removing the directory, even if it is empty. It might be surprising later that suddenly a directory has different permissions/mode/whatever and that happened because `CopyTree` just removed it and then recreated with different permissions.